go/sentry/tendermint/tendermint.go

```diff
@@ -0,0 +1,49 @@
+// Package tendermint implements the tendermint backed sentry backend.
```

Comment:
I think we don't need a "sentry backend" for each consensus backend. As long as `GetAddresses()` is exposed by the general `consensus.Backend` interface, this service can be totally consensus backend-agnostic.

I would just move this implementation to `go/sentry` directly (no `tendermint` directory needed) and change it so that instead of passing a `TendermintService` you pass a `consensus.Backend`.

Reply:
Agreed. Done.